hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -192,7 +191,7 @@ public static short getDefaultReplication(final FileSystem fs, final Path path)
     Method m = null;
     Class<? extends FileSystem> cls = fs.getClass();
     try {
-      m = cls.getMethod("getDefaultReplication", new Class<?>[] { Path.class });
+      m = cls.getMethod("getDefaultReplication", Path.class);
```

Comment:
The method `getDefaultReplication(Path)` has been a supported API in all the Hadoop versions we support for a very longs time. We should skip the reflection here.

Reply:
Removed the reflection part.